### ISSUE TITLE
How about not give validation set?

### ISSUE THREAD

**Issue by Leafaeolian** [ArcInstitute/state#248]
Does anyone know why, even though `val` isn't included in the `toml` file, `val_loss` still appears in the output of `best.ckpt` and `metric.csv`?

batch_time_avg,batch_time_cv_percent,batch_time_max,batch_time_max_min_ratio,batch_time_min,batches_per_second,cell_sets_per_sec,cumulative_flops,cumulative_flops_val_sync,epoch,flops_per_batch,mfu (%),step,train/gradient_norm,train_loss,val_loss
,,,,,,51.8590087890625,33262992883712.0,,0,665259868160.0,3.5937204360961914,49,23.468921661376953,7.575930118560791,
0.060975972563028336,243.69833374023438,1.1091744899749756,47.441566467285156,0.023379802703857422,16.39990234375,67.75698852539062,66525985767424.0,,0,665259868160.0,4.6954169273376465,99,5.657567024230957,3.4721455574035645,
0.052493028342723846,27.722122192382812,0.08761143684387207,3.334625005722046,0.026273250579833984,19.050148010253906,88.887939453125,99788976553984.0,,0,665259868160.0,6.15974760055542,149,4.48955774307251,2.8143489360809326,
0.05850774422287941,33.924869537353516,0.14755463600158691,5.344603061676025,0.027608156204223633,17.091754913330078,94.64436340332031,133051971534848.0,,0,665259868160.0,6.558655738830566,199,8.967851638793945,2.281151056289673,
0.06338430941104889,22.797931671142578,0.09306716918945312,2.2954726219177246,0.04054379463195801,15.776775360107422,52.40211486816406,166314974904320.0,,0,665259868160.0,3.631356716156006,249,3.624199151992798,2.2907562255859375,
0.04936159774661064,41.34526443481445,0.09861016273498535,4.21577262878418,0.023390769958496094,20.258665084838867,90.53112030029297,199577953107968.0,,0,665259868160.0,6.273616790771484,299,5.734337329864502,1.6571931838989258,
,,,,,,82.61815643310547,232840948088832.0,,1,,5.725265026092529,349,4.220790863037109,1.5956883430480957,
0.05736570432782173,25.690773010253906,0.0907888412475586,3.158219575881958,0.028746843338012695,17.432018280029297,66.84561920166016,266103943069696.0,,1,,4.632261276245117,399,4.231783866882324,1.7802619934082031,
0.056861571967601776,30.082307815551758,0.09827804565429688,3.7144556045532227,0.026458263397216797,17.58656883239746,88.58815002441406,299366938050560.0,,1,,6.138973236083984,449,6.116981029510498,1.6658754348754883,
0.058829475194215775,21.375911712646484,0.09664511680603027,2.206804037094116,0.04379415512084961,16.998281478881836,78.76543426513672,332629949808640.0,,1,,5.458279132843018,499,5.183351039886475,2.6412949562072754,
,,,,,,,,332629949808640.0,1,,,499,,,**3.832207441329956**👈
